twint/output.py

```diff
@@ -24,7 +24,10 @@ def is_tweet(tw):
 def _output(obj, output, config):
     if config.Output != None:
         if config.Store_csv:
-            write.Csv(obj, config)
+            try :
+                write.Csv(obj, config)
+            except Exception as e:
+                print("Error: " + str(e))
```

Comment:
What kind of errors are you expecting? 
If the file does not exist, then it will be created.
If the file already exists the data will be appended.

Errors that could be expected are external to Twint, such of these could be "permission errors" and so the user that's running the process can not write to the file or create a file in the directory.

Looking forward to better understand your point.